packaging/ubuntu-16.04/rules

```diff
@@ -315,8 +315,8 @@ override_dh_systemd_start:
 	# debhelper supporting --remaining like behaviour here would
 	# have been useful
 	# (Re-)start socket first
-	dh_systemd_start snapd.socket
+	dh_systemd_start --restart-after-upgrade snapd.socket
```

Comment:
Does hti still behave as expected when executed on 16.04?

Reply:
No idea, but I can try to add a spread test. I'm just not sure how to trick apt into believing that there's an upgrade available.

Reply:
It seems that the option is supported there too. I'm still working on a spread test, though.

Reply:
We could dpkg-buildpackage/gbp buildpackage a package on 16.04 and compare the maintainer scripts of the "official" deb and the version with this to double check that things really work as expected there.

Reply:
I added a spread test for the behavior now and also I think we want https://github.com/snapcore/snapd/pull/11702 here because right now I'm pretty confident that on 16.04 debhelper is being silly :)